lib/src/resources/sources.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:async';
+
+import 'package:stripe/messages.dart';
+
+import '../client.dart';
+import '_resource.dart';
+
+/// Source objects allow you to accept a variety of payment methods.
+/// They represent a customer’s payment instrument,
+/// and can be used with the Stripe API just like a Card object:
+/// once chargeable, they can be charged, or can be attached to customers.
+@Deprecated('''Stripe doesn't recommend using the deprecated Sources API.
+We recommend that you adopt the PaymentMethods API.
+This newer API provides access to our latest features and payment method types.
+''')
+class SourcesResource extends Resource<Subscription> {
```

Comment:
Please, check whether we use singular or plural names for resources. It seems like it should be `SourceResource`.